platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -31,6 +31,8 @@
 #define _ARM_CRYPTO_MAX_CIPHER_KEY_LENGTH      64
 #define _ARM_CRYPTO_MAX_AUTH_KEY_LENGTH        64
 #define _ARM_CRYPTO_MAX_IV_LENGTH              16
+#define _ARM_CRYPTO_MAX_DATA_LENGTH	       65536
+#define _ARM_CRYPTO_MAX_DIGEST_LENGTH          64
```

Comment:
I do not see why this is 64 instead of 16.

Reply:
I will make the change. Thanks!